hikari/impl/shard.py

```diff
@@ -741,6 +746,7 @@ async def _run_once(self) -> bool:
                     # to always kill our connection if it dies.
                     # We return True if zombied
                     if await heartbeat_task:
+                        now = date.monotonic()
```

Comment:
What is this for?

Reply:
`now` might be undefined, since it only gets defined in L730. This just makes sure its defined.